UnityProject/Assets/Scripts/UI/Systems/ServerInfoUILobby.cs

```diff
@@ -199,7 +205,7 @@ public override void Process(NetMessage msg)
 		{
 			ServerInfoLobbyMessageServer.Send(
 					SentByPlayer.Connection, ServerData.ServerConfig.ServerName,
-					ServerInfoUILobby.serverDesc, ServerInfoUILobby.serverDiscordID, ServerInfoUILobby.serverRules);
+					UIManager.Instance.ServerInfoUILobby.ServerDesc.text, UIManager.Instance.ServerInfoUILobby.ServerDiscordID, UIManager.Instance.ServerInfoUILobby.ServerRules.text);
```

Comment:
separate each argument into its own line.

I thought this was an insanely long chain of stuff at first glance.